modules/tfm/tfm/boards/partition/region_defs.h

```diff
@@ -60,22 +53,38 @@
 #define S_DATA_SIZE     (PM_TFM_SRAM_SIZE)
 #define S_DATA_LIMIT    (S_DATA_START + S_DATA_SIZE - 1)
 
+#define S_CODE_VECTOR_TABLE_SIZE (CONFIG_TFM_S_CODE_VECTOR_TABLE_SIZE)
+
+#if defined(NULL_POINTER_EXCEPTION_DETECTION) && S_CODE_START == 0
+/* If this image is placed at the beginning of flash make sure we
+ * don't put any code in the first 256 bytes of flash as that area
+ * is used for null-pointer dereference detection.
+ */
+#define TFM_LINKER_CODE_START_RESERVED (256)
+#if S_CODE_VECTOR_TABLE_SIZE < TFM_LINKER_CODE_START_RESERVED
+#error "The interrupt table is too short too for null pointer detection"
```

Comment:
```suggestion
#error "CONFIG_TFM_S_CODE_VECTOR_TABLE_SIZE must be greater than 255"
```

"too short too for null"

Reply:
Fixed the stray "too", this keeps text aligned with the duplicated TF-M region_defs error messages, which I don't care to update.